config/config.go

```diff
@@ -46,8 +46,10 @@ type EvmConfig struct {
 	RPCURL              string `mapstructure:"rpc_url"`
 	SupplierName        string `mapstructure:"supplier_name"`
 	BookingTokenAddress string `mapstructure:"booking_token_address"`
-	BookingTokenABIFile string `mapstructure:"booking_token_abi_file"`
 	BuyableUntilDefault uint64 `mapstructure:"buyable_until_default"`
+	CMAccountAddress    string `mapstructure:"cm_account_address"`
+	DomainName          string `mapstructure:"domain_name"`
+	DomainVersion       uint64 `mapstructure:"domain_version"`
```

Comment:
What are these for?

Reply:
I don't remember the intention, but they shouldn't be there. I will remove them